src/contracts/interfaces/IConquest.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity 0.7.4;
+
+/**
+ * @notice Keep track of players participation in Conquest and used to issue rewards.
+ * @dev This contract must be at least a TIER 1 owner of the silverCardFactory and
+ *      goldCardFactory.
+ */
+interface IConquest {
+
+  /***********************************|
+  |      Receiver Method Handler      |
+  |__________________________________*/
+
+  /**
+   * @notice Players entering conquest with conquest entry token
+   * @dev Payload is passed to and verified by onERC1155BatchReceived()
+   */
+  function onERC1155Received(
```

Comment:
Could be nicer to `interface IConquest is IERC1155TokenReceiver` instead of copy pasting them here if it's possible.